BaseApp/COD57000.TXT

```diff
@@ -0,0 +1,40 @@
+OBJECT Codeunit 57000 DotNet Add Text Encoding
+{
+  OBJECT-PROPERTIES
+  {
+    Date=18-06-14;
+    Time=16:39:00;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      Encoding@1000 : DotNet "'mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.Encoding";
```

Comment:
As per @grobyns comment, these wrapper should only every have 1 global variable. You can see more examples in COD3000..COD3024 in this repo.

Reply:
Ok, It's applied.